theatre/shared/src/getAllPossibleAssetIDs.ts

```diff
@@ -0,0 +1,22 @@
+import type Project from '@theatre/core/projects/Project'
+import {val} from '@theatre/dataverse'
+
+export default function getAllPossibleAssetIDs(project: Project) {
+  const sheets = Object.values(val(project.pointers.historic.sheetsById))
```

Comment:
I don't know why TS didn't catch this, but `val(...sheetsById)` could return `undefined`.